js/pulse.js

```diff
@@ -1,15 +1,29 @@
 /*
  * Pulse ad player ad manager
- *
- * version 1.0
  */
 
 (function(_, $)
 {
-
     var pulseAdManagers = {};
 
     OO.Ads.manager(function(_, $) {
+        function log() {
+            var args = Array.prototype.slice.call(arguments);
+            if(OO.Pulse) {
+                OO.Pulse.Utils.log.apply(null, args);
+            } else {
+                args.unshift('OO.Pulse: ');
+                console.log.apply(window.console, args);
+            }
+        }
+
+        var AD_MODULE_STATE = {
+            Uninitialized: 'uninitialized',
```

Comment:
Just a nitpicky comment: since this is "constants" object, perhaps the properties should also be all caps
(i.e. `UNINITIALIZED` as opposed to `Uninitialized`)